static/fetchToken.js

```diff
@@ -0,0 +1,22 @@
+document.querySelector('.login-form').addEventListener('submit', (e) => {
+
+    const username = document.querySelector('.login-form__input')
+
+    fetch('/login', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+            username:username
+        })
+    })
+    .then(response => response.json())
+    .then(data => {
+        console.log(data.token)
```

Comment:
Is console.log() for the token supposed to be here or can it be removed?

Reply:
I was actually supposed to delete the file... i am not storing the cookie that way anymore so is not needed